### ISSUE TITLE
Question: By using this, can we run shadcn/ui inside Svelte?

### ISSUE THREAD

**Issue by sonyarianto** [bfanger/svelte-preprocess-react#42]
By using this, can we run shadcn/ui inside Svelte?

**Comment by bfanger** [comment]
Yes, but It depends on which component on how easy it is to use. 
Some components such as `<Dialog/>` are using `React.Children.only` internaly and then you can use that component directly inside a svelte file and you'd need to write a custom wrapper component.

I'd recommend looking into https://shadcn-svelte.com/

**Comment by sonyarianto** [comment]
@bfanger thanks for the answer, ya the idea is to use native shadcn/ui inside the Svelte since that already very good and avoid using the porting version of shadcn-svelte. That's just an idea :)
